src/Chapter1.hs

```diff
@@ -620,7 +631,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = lastD + secondlastD
+  where
+    lastD = (abs n) `mod` 10
```

Comment:
Great!
You can notice that this is actually the `lastDigit` function from the previous task 🙂